controller/work_item_events.go

```diff
@@ -111,8 +101,8 @@ func ConvertEvent(appl application.Application, request *http.Request, wiEvent e
 			},
 		}
 	case workitem.SystemArea:
-		old := ConvertAreaSimple(request, wiEvent.Old)
-		new := ConvertAreaSimple(request, wiEvent.New)
+		old, _ := ConvertAreaSimple(request, wiEvent.Old)
```

Comment:
Why can't we add the links in the relationships?

Reply:
We cannot have links for *plural* objects but this is a single object

Reply:
@jarifibrahim yes, this is a single object but please note how we stuff that object into an array further below: `Data: []*app.GenericData{old},`. There simple is no way for us to put the link to an individual area in that array.

Reply:
Okay. Got that. We should've modelled `OldValue` and `NewValue` to be `RelationGeneric` instead of `RelationGenericList`.

Reply:
For now the UI expects a list always. And we use the list for single and multi-value relationships.